src/revChatGPT/V3.py

```diff
@@ -22,7 +22,7 @@ class Chatbot:
 
     def __init__(
         self,
-        api_key: str,
+        api_key: str = None,
```

Comment:
Api key can be included in config, so we shouldn't require it.

Reply:
Need checks to see if it's included in at least 1